custom/icds/messaging/custom_content.py

```diff
@@ -277,6 +278,10 @@ def aww_2(recipient, case_schedule_instance):
     return run_indicator_for_usercase(case_schedule_instance.case, AWWAggregatePerformanceIndicator)
 
 
+def phase2_aww_1(recipient, case_schedule_instance):
```

Comment:
we'll also need to add an entry in `settings.AVAILABLE_CUSTOM_SCHEDULING_CONTENT ` for this one

Reply:
https://github.com/dimagi/commcare-hq/commit/3663bdd3903ae373ae6fefe85ddf5a3bf95696cc